static/js/common.js

```diff
@@ -20,16 +20,16 @@ function password_quality(password, bar, password_field) {
         return undefined;
     }
 
-    var min_length = 6;
+    var min_password_length = 1;
```

Comment:
I just realized, we should change this to 0, to allow no password? It seems that would be the actual use case if you had no password requirement.